mmv1/products/datafusion/Instance.yaml

```diff
@@ -237,16 +249,59 @@ properties:
         description: |
           The IP range in CIDR notation to use for the managed Data Fusion instance
           nodes. This range must not overlap with any other ranges used in the Data Fusion instance network.
-        required: true
+        required: false
         immutable: true
       - !ruby/object:Api::Type::String
         name: 'network'
         description: |
           Name of the network in the project with which the tenant project
           will be peered for executing pipelines. In case of shared VPC where the network resides in another host
           project the network should specified in the form of projects/{host-project-id}/global/networks/{network}
-        required: true
+        required: false
+        immutable: true
+      - !ruby/object:Api::Type::Enum
+        name: 'connectionType'
+        description: |
+          Optional. Type of connection for establishing private IP connectivity between the Data Fusion customer project VPC and
+          the corresponding tenant project from a predefined list of available connection modes.
+          If this field is unspecified for a private instance, VPC peering is used.
+        values:
+          - :CONNECTION_TYPE_UNSPECIFIED
+          - :VPC_PEERING
+          - :PRIVATE_SERVICE_CONNECT_INTERFACES
+        required: false
+        immutable: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'privateServiceConnectConfig'
+        description: |
+          Optional. Configuration for Private Service Connect.
+          This is required only when using connection type PRIVATE_SERVICE_CONNECT_INTERFACES.
+        required: false
         immutable: true
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'networkAttachment'
+            description: |
+              Required. The reference to the network attachment used to establish private connectivity.
```

Comment:
Currently the field's description says it's required but the field isn't marked as required. This means [the generated documentation describes the field as both required and optional](https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-10969-old..auto-pr-10969#diff-980a1051f9f4b9f12e86b1fa3861a8e98253f6b2b1f60034ebf10bec0c51dd0fR382-R384). Could you please fix this discrepancy?

Reply:
I have modified the description